lib/optimizer/src/Transformer/AmpRuntimeCss.php

```diff
@@ -120,6 +120,7 @@ private function findAmpRuntimeStyle(Document $document, ErrorCollection $errors
         if (! $ampRuntimeStyle instanceof DOMElement) {
             $version = $this->configuration->get(AmpRuntimeCssConfiguration::VERSION);
             $errors->add(Error\CannotInlineRuntimeCss::fromMissingAmpRuntimeStyle($version));
+            return false;
```

Comment:
I opted for `false` because:
- that's what the comment claimed
- that can be checked for reliably, whereas `null` can always be the result of forgetting to set something.